src/flatbuffers/schema/README.md

```diff
@@ -0,0 +1,42 @@
+# GenAI Flatbuffer Schemas
+This directory contains [the GenAI Lora Parameter format schema](genai_lora.fbs) and [the generated C++ header file](genai_lora.fbs.h) for the
+Lora Parameters file format. This file format is defined as a portable intermediate between Numpy `npz` format that is produced
+by Olive when exporting Lora Parameters as external files.
+
+The format supports multiple Lora Parameters per file.
+
+[The GenAi Lora Parameter file format schema](genai_lora.fbs) uses the [FlatBuffers](https://github.com/google/flatbuffers) serialization library.
+
+Please do not directly modify the generated C++ header file for [The GenAi Lora Parameter file format]((genai_lora.fbs.h))
+or [the generated Python binding files](../genai_flatbuffers_py).
```

Comment:
Seems like the python binding was not checked in/does not exist? Can we add that as well?

Reply:
I will remove the reference to it. I do not plan on checking in python code for this.